spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -328,7 +328,7 @@ def current_build_number
           build_version
         else
           if candidate_builds.length > 0
-            candidate_builds.first.build_version
+            candidate_builds.sort_by(&:upload_date).last.build_version
```

Comment:
my ruby is ... lacking: is there a way to reverse sort this so you don't have to use  `.last` here?

Reply:
my rube is ... same. will something like https://stackoverflow.com/a/2642196/2380455 go fine?

Reply:
`sort_by().last` on the other side is the simplest and clearest 😃